lms/urls.py

```diff
@@ -237,6 +237,19 @@
     )
 
 if settings.COURSEWARE_ENABLED:
+    COURSE_URLS = patterns(
```

Comment:
Yes. I am fine if you choose to not do this. I would just like the debt to be tracked.